sanitizer/_geo_reverse.js

```diff
@@ -1,10 +1,9 @@
-var geo_common = require ('./_geo_common');
-var _ = require('lodash');
-var defaults = require('../query/reverse_defaults');
-var LAT_LON_IS_REQUIRED = true,
-    CIRCLE_IS_REQUIRED = false;
-
-const non_coarse_layers = ['venue', 'address', 'street'];
```

Comment:
this was unused

Reply:
Huh, I wonder when we last used it... We have long needed to improve reverse geocoding support for custom layers, as mentioned in https://github.com/pelias/api/issues/1161. Not something to solve in this PR though.